bats/tests/snapshots/create-use-snapshot.bats

```diff
@@ -82,6 +86,17 @@ local_setup() {
     done
 }
 
+@test "attempt to create a snapshot with an existing name is flagged and doesn't do factory-reset" {
+    # RD state is currently stopped, so best to shut it down
+    rdctl shutdown
+    assert_exists "$PATH_APP_HOME/rd-engine.json"
```

Comment:
Part of this is in the test title: `@test "... and doesn't do factory-reset"`

Good point on rd-engine.json -- changed the assertion to `$PATH_CONFIG_FILE`.

And I do the existence test before running `snapshot create` because if I want to verify that
the failed `snapshot create` doesn't delete it. I have to do an indirect test like this.